jobs/loc/TranslationsImportExport.yml

```diff
@@ -36,11 +36,12 @@ steps:
   inputs:
     locProj: 'jobs/loc/LocProject.json'
     outDir: '$(Build.ArtifactStagingDirectory)'
-    isCreatePrSelected: false
```

Comment:
You will want to leave this field as `false` if you are using: https://github.com/microsoft/vscode-makefile-tools/blob/main/translations_auto_pr.js

In cpptools and cmaketools, we check in the i18n files (which are more straight-forward to manually maintain, if necessary) to the repo.  I believe if you were to use the PR they generate, that would skip the `translates-import` script, and it would instead try to check in `xlf` files.  If you want to use to the same approach as cpptools and cmaketools, I would leave this as `false`.